app/models/PsaFSDetail.scala

```diff
@@ -192,26 +192,26 @@ object PsaChargeType extends Enumeration {
 
   sealed case class TypeValue(name: String, value: String) extends Val(name)
 
-  val aftInitialLFP: TypeValue = TypeValue("57001080", "Accounting for Tax late filing penalty")
-  val aftDailyLFP: TypeValue = TypeValue("57001091", "Accounting for Tax further late filing penalty")
-  val aft30DayLPP: TypeValue = TypeValue("57301080", "Accounting for Tax late payment penalty (30 days)")
-  val aft6MonthLPP: TypeValue = TypeValue("57301091", "Accounting for Tax late payment penalty (6 months)")
-  val aft12MonthLPP: TypeValue = TypeValue("57301092", "Accounting for Tax late payment penalty (12 months)")
-  val otc30DayLPP: TypeValue = TypeValue("57401080", "Overseas transfer charge late payment penalty (30 days)")
-  val otc6MonthLPP: TypeValue = TypeValue("57401091", "Overseas transfer charge late payment penalty (6 months)")
-  val otc12MonthLPP: TypeValue = TypeValue("57401092", "Overseas transfer charge late payment penalty (12 months)")
+  val aftInitialLFP: TypeValue = TypeValue("57001080", "Accounting for Tax Late Filing Penalty")
+  val aftDailyLFP: TypeValue = TypeValue("57001091", "Accounting for Tax Further Late Filing Penalty")
+  val aft30DayLPP: TypeValue = TypeValue("57301080", "Accounting for Tax 30 Days Late Payment Penalty")
+  val aft6MonthLPP: TypeValue = TypeValue("57301091", "Accounting for Tax 6 Months Late Payment Penalty")
+  val aft12MonthLPP: TypeValue = TypeValue("57301092", "Accounting for Tax 12 Months Late Payment Penalty")
```

Comment:
If these descriptions are displayed in the front-end then when we come to do Welsh translation it will cause a problem. Really we should have all content for display in the front-end. For now this is fine I guess but we do need to have a ticket to move this content somehow into the front-end.